core/server/worker/src/main/java/alluxio/worker/block/meta/StorageTier.java

```diff
@@ -99,7 +104,7 @@ private void initStorageTier()
       int mediumTypeindex = i >= dirMedium.length ? dirMedium.length - 1 : i;
       long capacity = FormatUtils.parseSpaceSize(dirQuotas[index]);
       try {
-        StorageDir dir = StorageDir.newStorageDir(this, i, capacity, reservedSpaceBytes,
+        StorageDir dir = StorageDir.newStorageDir(this, i, capacity, reservedBytes,
```

Comment:
Should we reserve the total tier reserve bytes in each directory in the tier?

Reply:
This is indeed per-dir reserved bytes.
There is no total reserved bytes defined since we want to be able to make progress on each directory.